Sources/KlaviyoCore/Models/APIModels/CreateEventPayload.swift

```diff
@@ -120,8 +120,8 @@ extension Dictionary where Key == String, Value == Any {
             "Device Model": context.deviceModel,
             "OS Name": context.osName,
             "OS Version": context.osVersion,
-            "SDK Name": __klaviyoSwiftName,
-            "SDK Version": __klaviyoSwiftVersion,
+            "SDK Name": environment.SDKName,
```

Comment:
I think we use version and sdk name in the user agent too - should it be updated there?

Reply:
Thanks for that callout. Yeah I missed that and updated it now.